xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -142,7 +142,14 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>require(['jquery', 'xwiki-meta'], function ($, xm) {
+      <code>define('like-button-title-keys', {
```

Comment:
Why not `like-translation-keys` or `xwik-like-messages` (no sure where have a rule about the naming of such components, cc @mflorea).
The current name feels too specific.

Reply:
Each l10n JavaScript module generates an additional HTTP request (for the REST API) to fetch the specified translation keys, so we should limit the number of such modules, i.e. have at most one for ``-ui`` module. In this case, as @manuelleduc suggested, we should have a single l10n JavaScript module for the like feature. As for the name, it should have the ``xwiki-`` prefix, as per https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HJavaScriptBestPractices , followed by the ``-ui`` module name, e.g. ``like`` and followed by ``-messages`` suffix (most used so far). This means ``xwiki-like-messages`` as @manuelleduc suggested.